static/app/components/searchQueryBuilder/hooks/useQueryBuilderState.tsx

```diff
@@ -52,6 +53,7 @@ type QueryBuilderState = {
    * This is the basic source of truth for what is currently being displayed.
    */
   query: string;
+  replacedRawSearchKey: boolean;
```

Comment:
Is there any way to avoid an extra state variable? I'd hate to add this if we don't have to